apps/checkout-app/backend/payments/types.ts

```diff
@@ -1,9 +1,11 @@
-import { OrderCreateFromCheckoutErrorCode } from "@/graphql";
+import { OrderCreateFromCheckoutErrorCode } from "@/checkout-app/graphql";
 
 type InternalErrorCodes =
   | "COULD_NOT_CREATE_ORDER_FROM_CHECKOUT"
   | "TOTAL_AMOUNT_MISMATCH"
-  | "UNKNOWN_PROVIDER";
+  | "UNKNOWN_PROVIDER"
+  | "MISSING_CHECKOUT_OR_ORDER_ID"
```

Comment:
No way we could specify this?

Reply:
Not really, because both are valid and if neither is passed we do not know which was intended to be passed.